### DISCUSSION TITLE
TimesFormer for NTU120?

### DISCUSSION THREAD

**Discussion by taliegemen** [open-mmlab/mmaction2#2072] [Q&A]
Hello, I am trying to use dividedST timesformer approach to train NTU120 dataset. The maximum accuracy that I have is around 0.74 top-1 whereseas resnet50slow model achieves around 0.85 performance. The network fully learns the training set (with more than 99 top-1) but overfits on validation set. What to do at this point?
